configs/mot/qdtrack/qdtrack_faster-rcnn_r50_fpn_4e_dancetrack.py

```diff
@@ -0,0 +1,127 @@
+_base_ = ['./qdtrack_faster-rcnn_r50_fpn_4e_crowdhuman_mot17-private-half.py']
+model = dict(
+    type='QDTrack',
+    detector=dict(
+        backbone=dict(
+            norm_cfg=dict(requires_grad=False),
+            style='caffe',
+            init_cfg=dict(
+                type='Pretrained', checkpoint='torchvision://resnet50')),
+        rpn_head=dict(bbox_coder=dict(clip_border=False)),
+        roi_head=dict(
+            bbox_head=dict(
+                loss_bbox=dict(type='L1Loss', loss_weight=1.0),
+                bbox_coder=dict(clip_border=False),
+                num_classes=1)),
+        init_cfg=dict(
+            type='Pretrained',
+            checkpoint=  # noqa: E251
+            'https://download.openmmlab.com/mmdetection/v2.0/faster_rcnn/faster_rcnn_r50_fpn_1x_coco-person/faster_rcnn_r50_fpn_1x_coco-person_20201216_175929-d022e227.pth'  # noqa: E501
+        )),
+    track_head=dict(
+        type='QuasiDenseTrackHead',
```

Comment:
I realized that it was a fault to set `./qdtrack_faster-rcnn_r50_fpn_4e_crowdhuman_mot17-private-half.py` as a base because it is based on `'../../_base_/datasets/mot_challenge.py'` while the dancetrack qdtrack config shoule be based on `'../../_base_/datasets/dancetrack.py'`. I think the right way should be creating a new config based on:
```
_base_ = [
    '../../_base_/models/faster_rcnn_r50_fpn.py',
    '../../_base_/datasets/dancetrack.py', '../../_base_/default_runtime.py'
]
```
Given the rule that different base config files should have no key conflict, the dancetrack qdtrack config file should inherit no config file using `'../../_base_/datasets/mot_challenge.py'`. To be precise, if it inherits `'../../_base_/datasets/dancetrack.py'` and `./qdtrack_faster-rcnn_r50_fpn_4e_mot17-private-half.py'` at the same time, it raises error 
```
KeyError: "Duplicate key is not allowed among bases. Duplicate keys: {'data_root', 'train_pipeline', 'img_norm_cfg', 'test_pipeline', 'data', 'dataset_type'}"
```